### ISSUE TITLE
aleph, compojure, and http-kit benchmarks are incorrect

### ISSUE THREAD

**Issue by yogthos** [TechEmpower/FrameworkBenchmarks#1894]
These benchmarks appear to have been created from the same template and use `def` to define queries, e.g:

```clojure
(def single-query-test-raw
  "Test 2: Single database query (raw)"
  (-> 1
      (run-queries-raw)
      (first)
      (ring-resp/response)))
```

This runs the code at compile time and binds the resulting value to the variable. This code isn't testing the performance at runtime.

**Comment by k-r-g** [comment]
Ouch, good catch. I'll create a PR to fix these.

**Comment by msmith-techempower** [comment]
Resolved with #1896
